public/forms/validation/form.ttl

```diff
@@ -101,50 +101,61 @@ fields:validatonSelectorF a form:Field ;
     ] ;
   form:displayType displayTypes:conceptSchemeSelector ;
   form:options """{"conceptScheme":"http://lblod.data.gift/concept-schemes/dde3d2a3-e848-47ea-ba44-0f2e565f04ab","searchEnabled":true}""" ;
-  sh:group ext:formFieldValidations .
+  sh:group ext:formFieldPg ;
+  form:hasConditionalFieldGroup fields:maxLengthValidationCFG .
+
+##########################################################
+#  Error message field
+##########################################################
+ext:validationResultMessageF a form:Field ;
+  sh:name "Error message" ;
+  sh:order 100 ; # I think we always want to show this below the conditional fields?
+  sh:path sh:resultMessage ;
+  form:displayType displayTypes:defaultInput;
+  sh:group ext:formFieldPg .
```

Comment:
I _think_ they all need to be the same PG, I think they didn't render when I used the second one, but maybe something else was wrong.

Reply:
Hmm, no that doesn't seem to be the case, so this can be improved since it now shows "Field" everywhere 😄